plane/api/endpoints/avatars.py

```diff
@@ -51,7 +72,7 @@ async def check_avatar(
 
         if not 0 <= threshold <= 1:
             raise ValueError(
-                'Parameter "threshold" must be of "float" type between 0 and 1'
+                'Parameter "threshold" must be of type "float" between 0 and 1'
```

Comment:
Should the numbers be enclosed in `""` or not?
I think not because Python's `!r` doesn't do that.